app/src/main/java/ru/androidacademy/gooseeye/fragments/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,67 @@
+package ru.androidacademy.gooseeye.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import android.widget.Toast
+import androidx.appcompat.widget.Toolbar
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentTransaction
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import ru.androidacademy.gooseeye.MainActivity
+import ru.androidacademy.gooseeye.models.ArtistInfo
+import ru.androidacademy.gooseeye.adapters.ArtistRecyclerAdapter
+import ru.androidacademy.gooseeye.R
+
+class FragmentMoviesDetails : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? = inflater.inflate(R.layout.fragment_movies_details, container, false)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setUpRecyclerViewArtist()
+        val backButton = view.findViewById<TextView>(R.id.btn_back)
+        backButton.setOnClickListener {
+            activity?.supportFragmentManager?.beginTransaction()?.apply {
+                remove(this@FragmentMoviesDetails)
+                setTransition(FragmentTransaction.TRANSIT_FRAGMENT_CLOSE)
+                commit()
```

Comment:
Better move function in to activity or use interface.

Reply:
I agree. One screen should not know about another screen directly.